app/tests/spec/lib/channels/receivers/postmessage.js

```diff
@@ -77,7 +77,7 @@ define(function (require, exports, module) {
         receiver.receiveEvent({
           data: {},
           origin: 'chrome://browser',
-          type: 'click'
```

Comment:
This typo is why the tests didn't catch the error before. `click` should have been `message`.